examples/mood-classifiers/src/featureExtraction.js

```diff
@@ -0,0 +1,45 @@
+importScripts('./lib/essentia.js-model.js');
+importScripts('./lib/essentia-wasm.module.js');
+
+const essentiaWASM = Module; // Module is EssentiaWASM, tweaked ES module version without exports
+const essentia = new essentiaWASM.EssentiaJS(false);
+
+function outputFeatures(f) {
+    postMessage({
+        features: f
+    });
+}
+
+function computeFeatures(audioData) {
+    const featuresStart = Date.now();
+    const frames = essentia.FrameGenerator(audioData, 512, 256);
+    const framesLength = frames.size();
+
+    const features = {
+        melSpectrum: [],
+        batchSize: 0,
+        melBandsSize: 96,
+        patchSize: 187
+    }; 
+    for (let i = 0; i < framesLength; i++) {
+        const spectrum = essentia.TensorflowInputMusiCNN(frames.get(i));
```

Comment:
Is there any particular reason why to use the `TensorflowInputMusiCNN` algorithm directly instead of the `EssentiaTFInputExtractor` wrapper function?

Reply:
Yes, in this case it saves me an `vectorToArray` conversion and then (internally in `EssentiaTFInputExtractor`) an `arrayToVector` conversion. Since `FrameGenerator` outputs VectorFloats and `TensorflowInputMusiCNN` takes VectorFloats too, I thought it was faster this way...

Reply:
I see. we should then add an option to also pass `VectorFloat` along with `Float32Array` type [here](https://github.com/MTG/essentia.js/blob/baa1522af9f0933121a56efcd65cd88234f94215/src/typescript/machinelearning/tfjs_input_extractor.ts#L168) in future (I will create an issue). 
For the sake of consistency, I think we can use the `EssentiaTFInputExtractor` here and also its not a real-time use case. We can fix this double conversion overhead internally as I mentioned above. Also, there is an `computeFrameWise` method to do offline extraction.